website/src/pages/SearchPage/SearchPage.tsx

```diff
@@ -42,27 +45,35 @@ export default function SearchPage() {
     <>
       <Hero setGitRef={setGitRef} />
 
-      {searchError && <div className="text-red-500 text-center my-2">{searchError}</div>}
+      {searchError && (
+        <div className="text-red-500 text-center my-2">{searchError}</div>
+      )}
 
       {isSearchLoading && (
         <div className="flex my-10 justify-center items-center">
           <RingLoader loading={isSearchLoading} color="#E77002" size={300} />
         </div>
       )}
 
-      {!isSearchLoading && dataSearch && (
+      {!isSearchLoading && dataSearch && dataSearch.length > 0 && (
```

Comment:
The `dataSearch.length > 0` is a good idea and fix. Let's actually duplicate this for the `/compare` page.